src/error/AbstractCydranError.spec.ts

```diff
@@ -0,0 +1,47 @@
+import {assert} from "chai";
+import {describe, it, xit} from "mocha";
+import {CydranError, NEWI} from "./AbstractCydranError";
+
+class TestError extends CydranError {
+
+	constructor(msg: string, reps?: any) {
+		if(!new.target) throw (new.target + NEWI);
```

Comment:
1. Curly braces are missing on conditional
2. Missing space between if and paren

Reply:
Done.